### DISCUSSION TITLE
How to - Command Menu

### DISCUSSION THREAD

**Discussion by jlorezz** [vercel/community#170] [Show & Tell]
What's the best way to approach building a Command Menu with shortcut etc? Using nextjs and tailwind rn?
how would I get the shortcuts to work/

**Comment by Pieparker** [comment]
Your question appears to be lacking quite a bit of context! Is there any chance you could elaborate on your question with examples or more detail?

**Comment by Arihantar** [comment]
Are you referring to browsers (dev tools) command menu or for the vscode or similar editors command menu?

**Reply by jlorezz** [reply]
I am talking about (as an example the web command menu (vercel)) you can press command k and have a menu to work with

**Comment by IlirBajrami** [comment]
I think this is what you need: https://kbar.vercel.app/
